opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5-interface.adoc

```diff
@@ -0,0 +1,124 @@
+
+[[telemetryd-netflow5-protocol]]
+==== NetFlow Version 5
+
+link:https://www.cisco.com/c/en/us/td/docs/net_mgmt/netflow_collection_engine/3-6/user/guide/format.html[NetFlow] is an application that provides the ability to collect IP network traffic from network devices and administrators can determine and analyze the network communication.
+This protocol allows to process and persist NetFlow v5 packets with _{opennms-product-name}_.
+Data is persisted in link:https://www.elastic.co/products/elasticsearch[ElasticSearch] and can be analyzed with link:https://www.elastic.co/products/kibana[Kibana].
+
+IMPORTANT: The NetFlow v5 requires a running ElasticSearch 5.6+ to persist the NetFlow data.
+
+By default the persistence is not installed by default and can be enabled through the Karaf Console with:
+
+[source]
+----
+$ ssh -p 8101 admin@localhost
+...
+admin@opennms()> feature:install flows
+admin@opennms()> config:property-set name Netflow-5
```

Comment:
These additional properties are not required for the `flows` feature. They are ignored in OpenNMS, and only used on Minion.